features/step_definitions/ocm/ocm.rb

```diff
@@ -1,18 +1,27 @@
-Given /^I open ocm portal as an? #{WORD} user$/ do |usertype|
+Given /^I open ocm portal with #{WORD}$/ do |usertype|
```

Comment:
Why do we change wording here? For me original wording is more meaningful. Could you give example of why it needs to be like this?

Reply:
We have some finished cases using 'I open ocm portal with' step. So I change it to avoid updating all the finished cases.